src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20160808_remove_workflow_failure_procedure.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="dummy" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet logicalFilePath="dummmy" author="ansingh" id="remove-workflow-failure-procedure-1">
+        <sql stripComments="true">
+            DROP PROCEDURE IF EXISTS migrate_workflow_failures;
```

Comment:
in general i'd say these types of things probably shouldn't be drop _if exists_ because if for some reason it doesn't exist, it will suppress the error and we won't actually know if this change succeeded or not